utils/iputils.go

```diff
@@ -79,6 +79,15 @@ func IsLocalIP(ip string, addrs *[]net.Addr) bool {
 	return false
 }
 
+func GetLocalIP(addrs *[]net.Addr) net.IP {
+	for _, address := range *addrs {
+		if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() && ipnet.IP.To4() != nil {
```

Comment:
what if the host have mutil interface, we do better compare the ip net cidr with the master0ip to select the right one.